Sources/SWBCore/BuildRequestContext.swift

```diff
@@ -134,8 +134,8 @@ extension BuildRequestContext {
         }
     }
 
-    /// Compute output paths for a list of source files in a specific target.
-    public func computeOutputPaths(for inputPaths: [Path], workspace: Workspace, target: BuildRequest.BuildTargetInfo, command: BuildCommand, parameters: BuildParameters? = nil) -> [String] {
+    /// Compute output paths for a source file in a specific target. There may be multiple results if the build is a multi-arch build.
+    public func computeOutputPaths(for inputPath: Path, workspace: Workspace, target: BuildRequest.BuildTargetInfo, command: BuildCommand, parameters: BuildParameters? = nil) -> [String] {
```

Comment:
This is no longer a many-to-many because that leaves the caller unable to distinguish which outputs corresponded to which inputs.